static/app/views/performance/traceDetails/index.tsx

```diff
@@ -164,4 +187,4 @@ class TraceSummary extends Component<Props> {
   }
 }
 
-export default withOrganization(withApi(TraceSummary));
+export default React.memo(withOrganization(withApi(TraceSummary)));
```

Comment:
I don't know if this is actually going to help for any performance gains because the props for `TraceSummary` are likely to change so much. React.memo by default compares props by reference so any objects that are passed down are likely going to trigger a re-render

I think there's a bit of memory overhead when using `React.memo` so maybe we should just remove it for now unless we know it's really helping

Reply:
Oops, yeah this should not be here. It got left behind from the trial and error phase. I'll remove it.